### PR TITLE
[1LP][RFR] Verify RBAC User/Group/Role is highlighted in accordion when Details view is displayed

### PR DESCRIPTION
**Pull request by LandoCalrizzian**
See https://bugzilla.redhat.com/show_bug.cgi?id=1498075

{{pytest: -v -k 'test_group_crud or test_user_crud or test_role_crud or test_delete_default_group or test_delete_default_roles or test_edit_default_group or test_edit_default_roles'}}

### CODE REVIEW

File: cfme/configure/access_control.py [hunk 1]
```diff
@@ -77,7 +77,9 @@ class DetailsUserView(ConfigurationView):
     def is_displayed(self):
         return (
             self.title.text == 'EVM User "{}"'.format(self.context['object'].name) and
-            self.accordions.accesscontrol.is_opened
+            self.accordions.accesscontrol.is_opened and
+            (self.accordions.accesscontrol.tree.selected_item.text ==
```
Comment (mkoura) [inline cfme/configure/access_control.py]: No need for extra parentheses, indentation is enough to visually connect the expression on two lines.
The ``return`` statement already has parentheses so additional ones are not needed for line continuation.

File: cfme/configure/access_control.py [hunk 2]
```diff
@@ -955,7 +959,9 @@ class DetailsRoleView(RoleForm):
     def is_displayed(self):
         return (
             self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Role "{}"'.format(self.context['object'].name)
+            self.title.text == 'Role "{}"'.format(self.context['object'].name) and
+            (self.accordions.accesscontrol.tree.selected_item.text ==
+                self.context['object'].name)
```
Comment (mfalesni) [inline cfme/configure/access_control.py]: **REQUIRED**: I would probably replace `.tree.selected_item.text` with `.tree.currently_selected[-1]` as by doing `.text` you bypass widgetastic browser. `currently_selected` returns a list or None if nothing selected.

Comment (mfalesni) [reply cfme/configure/access_control.py]: And you could actually check the whole tree path in there to check you indeed are where you should be, but that is optional in this case.

File: cfme/configure/access_control.py (hunk 1 above)
Comment (LandoCalrizzian) [reply cfme/configure/access_control.py]: Updated to remove the parenthesis. It was originally copy/pasted from another line of code that was longer than 100 chars which used them to wrap it.

File: cfme/configure/access_control.py (hunk 2 above)
Comment (LandoCalrizzian) [reply cfme/configure/access_control.py]: @mfalesni Updated to use tree.currently_selected w/ a comment to explain the use of the last item index.

**Pull request merged**